mac

```diff
@@ -166,29 +166,62 @@ brew_install_or_upgrade 'hub'
 # shellcheck disable=SC2016
 append_to_file "$HOME/.zshrc" 'eval "$(hub alias -s)"'
 
+
+fancy_echo 'Checking on Node.js installation...'
+
 if ! brew_is_installed "node"; then
-  if ! command -v n > /dev/null; then
-    fancy_echo 'Installing n and latest Node.js and NPM...'
-    curl -L http://git.io/n-install | bash -s -- -y latest
+  if command -v nvm > /dev/null; then
+    if ! command -v n > /dev/null; then
+      fancy_echo 'Installing n and latest Node.js and NPM...'
+      curl -L http://git.io/n-install | bash -s -- -y latest
+    else
+      fancy_echo 'Updating n...'
+      n-update -y
+    fi
   else
-    fancy_echo 'Updating n...'
-    n-update -y
+    fancy_echo 'nvm detected.  Skipping...'
   fi
+else
+  brew_install_or_upgrade 'node'
 fi
 
-brew_install_or_upgrade 'python3'
+fancy_echo '...Finished Node.js installation checks.'
+
+
+fancy_echo 'Checking on Python installation...'
+
+if ! brew_is_installed "python3"; then
```

Comment:
Derp! I forgot that the point was to install python with pyenv. I was expecting to have Python version 3 installed, but I was checking by looking for the `python3` brew. Now I understand that I need to do `pyenv versions`, and I see 3.5.1 as expected.